src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/ToDownloadProcessor.scala

```diff
@@ -68,8 +79,12 @@ trait ToDownloadProcessor extends BasicReaders with ScorexLogging {
       case _ if !isHeadersChainSynced || !nodeSettings.verifyTransactions =>
         // do not download full blocks if no headers-chain synced yet and suffix enabled or SPV mode
         Map.empty
+      case Some(fb) if fb.height < (estimatedTip.getOrElse(0) - 128) =>
```

Comment:
This would be good to have as function `fb.height < (estimatedTip.getOrElse(0) - 128)` for it to be self-explanatory

Reply:
done